src/constants.ts

```diff
@@ -23,9 +23,9 @@ export const mergeableImportFlavors = [
     importFlavorType,
 ] as const;
 
-export const BUILTIN_MODULES_REGEX_STR = `^(?:node:)?(?:${builtinModules.join(
+export const BUILTIN_MODULES_REGEX_STR = `(^(?:node:)?(?:${builtinModules.join(
     '|',
-)})$`;
+)})$)|(^(?:node:test)$)`; // NOTE: 'test' is not included in `builtinModules` so explicitly check for "node:test"
```

Comment:
```suggestion
)})$)|(^node:test$)`; // NOTE: 'test' is not included in `builtinModules` so explicitly check for "node:test"
```

I think this might be a simpler regexp?

Reply:
Ah, yup, that seems to work. Updating! (RegEx is one of my gaps I still need to work on 😅)